src/pages/docker/configure/configuration-sources.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Configuration sources
+description: See an overview about the different configuration sources for the Cloud Docker for Commerce tool.
+---
+
+# Configuration sources
+
+You can create the Docker Compose configuration to build and deploy the Docker containers for your Adobe Commerce project from the following sources:
+
+-  [Adobe Commerce on cloud infrastructure project configuration files](#cloud-configuration-for-commerce) for Cloud projects
+-  [Unified configuration](#unified-configuration) (`.magento.docker.yml`) for On-premises projects
+-  [CLI configuration](#cli-configuration) using `ece-docker build:compose` command options
+-  [Custom Docker Compose configuration file](custom-docker-compose.md) supports installation for both Cloud and On-premises projects
+
+<InlineAlert variant="info" slots="text"/>
+
+When you build the Docker Compose configuration file, the `ece-docker build:compose` command overwrites the existing `docker-compose.yml` configuration file. You can save customizations for the Docker Compose configuration in a `docker-compose.override.yml` file. If the `docker-compose.override.yml` file is present, then the override configuration merges with the base configuration. See [Override configuration](../quick-reference.md#override-configuration).
+
+## Cloud configuration for Commerce
+
+You need the following project configuration files to emulate a Cloud project in the Docker environment.
+
+-  [.magento.app.yaml][application]
+-  [.magento/services.yaml][services]
+
+Typically, these files supply the configuration settings in the generated `docker-compose.yml` file when you build and deploy a Cloud Docker environment from an Adobe Commerce on cloud infrastructure project directory.
+
+## Unified configuration
+
+If you do not have or want to use the Adobe Commerce on cloud infrastructure configuration files, Cloud Docker for Commerce supports a unified configuration file, `.magento.docker.yml`.
+
+This configuration file includes the following sections:
+
+```conf
+services: [] # List of services
+variables: [] # List of environment variables
+hooks: [] # List of available hooks
+mounts: [] #l List of available mounts
+```
+
+### Services
+
+The `services` section specifies the services configuration for the Docker environment with `version` and `enabled` fields.
+
+```yaml
+services:
+  php:
+    version: "7.3"
+    enabled: true|false
+```
+
+-  `services` can include `php`, `mysql`, `redis`, `elasticsearch`, `rabbitmq`, `cron`, and so on
+-  `version` specifies a [supported service version][Service configuration options]. The version must be compatible with the Adobe Commerce version you deploy.
+-  `enabled` defaults to `true` if not set
+
+#### PHP service
```

Comment:
Not necessary for this initial migration, but it might be interesting to experiment with a [tabbed code block](https://github.com/adobe/aio-theme/#code-block) format for the different YAML examples, where "heading" = "services, hooks, mounts, etc".

Reply:
I absolutely want to do something like that. Adding it to the list.